xml/System.Net.Quic/QuicStream.xml

```diff
@@ -553,6 +553,7 @@ Equivalent to using <xref:System.Net.Quic.QuicStream.WriteAsync(System.ReadOnlyM
         <summary>To be added.</summary>
         <returns>To be added.</returns>
         <remarks>To be added.</remarks>
+        <inheritdoc cref="M:System.Net.Quic.QuicStream.ReadAsync(System.Memory{{System.Byte}},System.Threading.CancellationToken)" />
```

Comment:
This is wrong. This should be pointing to the base type and should not have the brackets escaped. I'm fixing it in the tool.

Reply:
Oh, it's not a bug in the tool. This is how it was defined in triple slash:

https://github.com/dotnet/runtime/blob/8a4de2ebe5bd13f3622da314e6c137f8b028e3ee/src/libraries/System.Net.Quic/src/System/Net/Quic/QuicStream.cs#L236

@ManickaP did you intend to inherit the documentation from [Stream.ReadAsync](https://github.com/dotnet/runtime/blob/8a4de2ebe5bd13f3622da314e6c137f8b028e3ee/src/libraries/System.Private.CoreLib/src/System/IO/Stream.cs#L310) as in my suggestion below, or did you intend something else?:

```suggestion
        <inheritdoc cref="M:System.IO.Stream.ReadAsync(System.Memory{{System.Byte}},System.Threading.CancellationToken)" />
```

Note: [Stream.ReadAsync is properly documented in MS Docs](https://docs.microsoft.com/en-us/dotnet/api/system.io.stream.readasync?view=net-6.0#system-io-stream-readasync(system-memory((system-byte))-system-threading-cancellationtoken)), so inheritdoc would still work, if that was your intention (and I change the cref).